src/components/AreaCard/AreaCard.test.js

```diff
@@ -4,25 +4,63 @@ import '@testing-library/jest-dom/extend-expect';
 import { BrowserRouter as Router, Link } from "react-router-dom";
 import AreaCard from './AreaCard';
 
-describe("AreaCard", () => {
-  it("should display the area information", () => {
-    const { getByText } = render(<Router>
+describe("AreaCard - Listing Information", () => {
+  let utils;
+
+  beforeEach(() => {
+    utils = render(<Router>
         <AreaCard
           id={5}
           key={5}
           name={"River North"}
           shortname={"RiNo"}
           about={"RiNo is a burgeoning area with ..."}
         />
+      </Router>)
+  });
+
+  it("should display the area Name", () => {
+    const { getByText } = utils;
+    const nameElement = getByText("River North");
+
+    expect(nameElement).toBeInTheDocument();
+  })
+
+  it("should display the area Short Name", () => {
+    const { getByText } = utils;
+    const shortNameElement = getByText(`(RiNo)`);
+
+    expect(shortNameElement).toBeInTheDocument();
+  })
+
+  it("should display the area About", () => {
+    const { getByText } = utils;
+    const aboutElement = getByText("RiNo is a burgeoning area with ...");
+
+    expect(aboutElement).toBeInTheDocument();
+  })
+
+})
+
+describe("AreaCard - Overall Card", () => {
```

Comment:
I'm not sure the difference between "listing information" and "overall card" here.

Reply:
Listing information is just testing individual properties, overall card is testing if all properties are rendering correct at once.  Do you want me to remove the Overall Card test?

Reply:
Nothing here is happening async, so I think just the above tests cover everything.